### PR TITLE
Minor changes to the reported vs genetic checks AVRO models

### PR DESCRIPTION
**Pull request by mbleda**

### CODE REVIEW

File: schemas/IDLs/org.gel.models.metrics.avro/1.1.0/ReportedVsGenetic.avdl [hunk 1]
```diff
@@ -59,6 +59,10 @@ protocol ReportedVsGenetic {
         */
         string runDate;
         /**
+        Genome assembly
+        */
+        string genomeAssembly;
```
Comment (greglever) [inline schemas/IDLs/org.gel.models.metrics.avro/1.1.0/ReportedVsGenetic.avdl]: For your own sanity I would recommend making this an `enum` and not a `string`.
See for example:
https://github.com/genomicsengland/GelReportModels/blob/8c8ff2b5006ba45d7d62b8e6519c736c12a55382/schemas/IDLs/org.gel.models.cva.avro/0.4.0/Variant.avdl#L11

**Comment by greglever** [comment]
otherwise looks good:
```
greg@greg-Latitude-E7470:~/gel/GelReportModels$ sudo ./build_models 
[sudo] password for greg: 
Sending build context to Docker daemon    339MB
Step 1/17 : FROM ubuntu:xenial
 ---> f7b3f317ec73
Step 2/17 : RUN apt-get update
 ---> Using cache
 ---> 7251abd4a51b
Step 3/17 : RUN apt-get install -y build-essential software-properties-common python-software-properties
 ---> Using cache
 ---> 3fba0b4694c7
Step 4/17 : RUN apt-get update
 ---> Using cache
 ---> 9129c9a7018e
Step 5/17 : RUN add-apt-repository ppa:webupd8team/java && apt-get update
 ---> Using cache
 ---> 4e9ef727496c
Step 6/17 : RUN echo "oracle-java8-installer shared/accepted-oracle-license-v1-1 select true" | debconf-set-selections
 ---> Using cache
 ---> 238c3bb2d4c0
Step 7/17 : RUN echo "oracle-java8-installer shared/accepted-oracle-license-v1-1 seen true" | debconf-set-selections
 ---> Using cache
 ---> 62ca13f4fb25
Step 8/17 : RUN apt-get update
 ---> Using cache
 ---> 38062ea8af8c
Step 9/17 : RUN apt-get install -y oracle-java8-installer maven     python-dev python-pip python-virtualenv     libsasl2-dev libldap2-dev libssl-dev     npm nodejs nodejs-legacy &&     npm install avrodoc -g
 ---> Using cache
 ---> 6d4c975d8338
Step 10/17 : ENV PYTHONUNBUFFERED 1
 ---> Using cache
 ---> 4576a7faf85f
Step 11/17 : RUN mkdir /gel
 ---> Using cache
 ---> f0b9d1acbf32
Step 12/17 : RUN mkdir /gel/GelReportModels
 ---> Using cache
 ---> c125ed46d100
Step 13/17 : WORKDIR /gel
 ---> Using cache
 ---> b82b3f8c1158
Step 14/17 : ADD . /gel/GelReportModels

[...]

INFO:root:Build/s finished succesfully!
Removing intermediate container f68f3a856323
 ---> 301dec678abc
Successfully built 301dec678abc
Successfully tagged gel:latest

root@5fd69605828b:/gel/GelReportModels# python -m unittest discover
[...]
----------------------------------------------------------------------
Ran 125 tests in 13.578s

OK
root@5fd69605828b:/gel/GelReportModels# 
```

File: schemas/IDLs/org.gel.models.metrics.avro/1.1.0/ReportedVsGenetic.avdl (hunk 1 above)
Comment (mbleda) [reply schemas/IDLs/org.gel.models.metrics.avro/1.1.0/ReportedVsGenetic.avdl]: I think it's a good idea. I will make it enum.
Thanks!

**Comment by mbleda** [comment]
Changes to the genomeAssembly enum pushed :)

**Comment by priesgo** [comment]
@mbleda a bit late, but why so many nullable `sampleIds`? Are we expecting data without `sampleIds`? I remember we were looking to have `sampleIds` in the family in order to support families with missinng members, but in elements with just one `sampleId` shouldn't it be there?

**Pull request merged**